docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   web:
     image: 613148481050.dkr.ecr.us-west-2.amazonaws.com/openduck:latest
     build: .
-    command: bash -c "python setup.py develop && aws s3 cp --recursive s3://uberduck-models-us-west-2/prototype models && uvicorn openduck_py.routers.main:app --reload --host 0.0.0.0 --port 8000"
+    command: bash -c "python setup.py develop && aws s3 cp --recursive s3://uberduck-models-us-west-2/prototype/download models && uvicorn openduck_py.routers.main:app --reload --host 0.0.0.0 --port 8000"
```

Comment:
i am once again asking if `aws s3 sync` wouldn't be better

Reply:
so true